### ISSUE TITLE
[BlenderBIM/IfcOpenShell] IFC file created with BlenderBIM or IfcOpenShell -> https://validate.buildingsmart.org/ does not recognize Authoring application 

### ISSUE THREAD

**Issue by Martin15135215** [IfcOpenShell/IfcOpenShell#2526]
So i created a ifc file like in this great beginner tutorial https://blenderbim.org/docs/users/creating_an_ifc_model.html#creating-a-single-object

And then i uploaded the created ifc file to https://validate.buildingsmart.org/ and it did not recognize Authoring application:

![image](https://user-images.githubusercontent.com/110968398/197402069-714e597e-2b6e-489e-8d9e-f56540e8a6b4.png)

When i create a ifc file with just IfcOpenShell with this awesome tutorial https://blenderbim.org/docs-python/ifcopenshell-python/code_examples.html#create-a-simple-model-from-scratch

and did the same like above and it also did not recognize Authoring application, in this case IfcOpenShell

![image](https://user-images.githubusercontent.com/110968398/197402403-96e28ce9-710f-4da4-8495-dde536806bf7.png)

Where lies the problem? By bSI?

**Comment by Moult** [comment]
I don't think this is a problem. I assume it's detecting IfcApplication, and in that case IfcApplication only exists if there is owner history tracking. Owner history tracking is optional in IFC4, and not enabled by default (if you want it, you can add a person / user / and set which one to use for owner histories in the project setup panel).

If you create an IFC2X3 project, where owner histories are mandatory, you should see the application.

**Comment by Martin15135215** [comment]
i thought it takes the information from the header section:
![image](https://user-images.githubusercontent.com/110968398/197460824-956caeea-9825-4057-85b7-f623aacec8db.png)

![image](https://user-images.githubusercontent.com/110968398/197460637-9cb55a80-046c-4548-a6f9-ea25c7e945bc.png)

**Comment by Martin15135215** [comment]
and one time the MVD was detected and one time not

**Comment by Martin15135215** [comment]
In the IFC file created by BlenderBIM, there is no space between `ViewDefinition` and `[DesignTransferView]`

I added the space with a text editor and then uploaded it to bSI:
![image](https://user-images.githubusercontent.com/110968398/197469099-2963f80e-6151-4199-a04b-0424aa536256.png)

Now it  detected the MVD used by BlenderBIM

**Comment by Moult** [comment]
Yeah I'd question that too, according to the official implementation guide by bSI for the header, there should be no space. See this document https://standards.buildingsmart.org/documents/Implementation/ImplementationGuide_IFCHeaderData_Version_1.0.2.pdf available on this page: https://technical.buildingsmart.org/resources/ifcimplementationguidance/

So ... I guess we wait for bSI to confirm?

**Comment by Moult** [comment]
Closing as not a bug with IfcOpenShell.
